gui/wxpython/startup/guiutils.py

```diff
@@ -143,3 +187,185 @@ def mapset_exists(database, location, mapset):
     if os.path.exists(mapset_path):
         return True
     return False
+
+
+def location_exists(database, location):
+    """Returns True whether location path exists."""
+    location_path = os.path.join(database, location)
+    if os.path.exists(location_path):
+        return True
+    return False
+
+
+def create_mapset_interactively(guiparent, grassdb, location):
```

Comment:
> Can you please take these new functions, paste them to a separate file, apply Black, and paste them back here? Perhaps check with Flake8 with default settings while you are at it. I see issues here and there, but this will likely just solve them automatically.

What do you mean by applying Black? I have checked Flake8 and it is alright...